src/pages/admin/Admin/AdminList/index.js

```diff
@@ -93,25 +93,28 @@ const AdminList = () => {
       return (
         <tr key={idx} className={style.tableDataRow}>
           <td className={style.tableData}>{admin.id}</td>
+          <td className={`${style.tableData}`}>
+            <Button buttonLabel="delete" buttonSize="sm" outline={true}/>
+          </td>
           <td className={style.tableData}>{admin.name}</td>
           <td className={style.tableData}>{admin.email}</td>
-          <td className={`${style.tableData} ${style.buttonColumn}`}>
-            <RiDeleteBin2Fill size="25px" color="#db7771" />
-          </td>
         </tr>
       );
     });
   };
 
   return (
     <div className={style.mainContent}>
-      <div>
+      <div className={style.headerTittleStyle}>
         <h1 className={style.pageTitle}>Admin Accounts</h1>
+        <Link to="/admin/create-admin-account">
+          <Button buttonLabel = "Add an Admin" buttonSize="def"/> 
+        </Link>
       </div>
       <Card className={style.card}>
         <Card.Header className={style.cardHeader}>
           <Form className={style.searchSection} onSubmit={onSearchSubmit}>
-            <div className={style.searchInput}>
+            {/* <div className={style.searchInput}>
               <Form.Control
                 className={style.searchBar}
                 type="text"
```

Comment:
is this still being used? please delete if not used

Reply:
That can be used in the future sir. in implementing reusable search bar.